piquasso/instructions/gates.py

```diff
@@ -651,6 +651,41 @@ def __init__(self, p: float) -> None:
         )
 
 
+class CubicPhase(Gate):
+    r"""Cubic Phase gate.
+
+    The definition of the Cubic Phase gate is
+
+    .. math::
+        \operatorname{CP}(\gamma) = e^{i \hat{x}^3 \frac{\gamma}{3 \hbar}}
+
+    The Cubic Phase gate transforms the annihilation operator as
```

Comment:
"transforms the annihilation operator" -> "written in terms of the annihilation operator"?